source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -162,6 +162,14 @@ Then import the JavaScript file before the closing `</body>` tag of your HTML pa
 </body>
 ```
 
+Errors from components will be logged in the browser's console.
+
+For example, when:
+
+* GOV.UK Frontend is not supported in the current browser
+* Component templates have missing changes from our release notes
+* Component JavaScript configuration does not match our documentation
+
```

Comment:
I like the new addition regarding execution stopping, but fixing errors applies to this section too?

```suggestion
* Component JavaScript configuration does not match our documentation

You must check your application works runs without errors or some components will not work correctly.
```

Reply:
I've used "should" here and "must" below